core/src/test/java/greencity/service/impl/FactOfTheDayTranslationServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+package greencity.service.impl;
+
+
+import greencity.ModelUtils;
+import greencity.dto.factoftheday.FactOfTheDayTranslationDTO;
+import greencity.entity.FactOfTheDayTranslation;
+import greencity.exception.exceptions.NotFoundException;
+import greencity.repository.FactOfTheDayTranslationRepo;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.modelmapper.ModelMapper;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+class FactOfTheDayTranslationServiceImplTest {
+
+    private FactOfTheDayTranslationServiceImpl factOfTheDayTranslationService;
+    @Mock
+    private FactOfTheDayTranslationRepo factOfTheDayTranslationRepo;
+    @Mock
+    private ModelMapper modelMapper;
+
+    @BeforeEach
+    void init() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
Please see 
- changes in https://github.com/ita-social-projects/GreenCity/pull/1353/files
- conversation in https://github.com/ita-social-projects/GreenCity/pull/1353/files/fd146c507103b8713a72173d64e2c9df32da32fb#diff-b727cc1b5710fa06c4f437b8b7413278 (click on Show outdated)

You will need:
- make fields final, remove `Autowired` annotation
- replace constructor by `AllArgsConstructor` annotation
- use `InjectMocks` annotation and remove `init` method